src/helper/helper.js

```diff
@@ -183,26 +184,24 @@ class DataLayerHelper {
   process() {
     if (this.processed_) {
       logError(`Process has already been ran. This method should only ` +
-        `run a single time to prepare the helper.`, LogLevel.ERROR);
+          `run a single time to prepare the helper.`, LogLevel.ERROR);
     }
 
-    // Process the existing/past states.
-    this.processStates_(this.dataLayer_, !(this.listenToPast_));
-
     // Register a processor for set command.
     this.registerProcessor('set', function() {
-      const model = this;
-
+      let toMerge = {};
       if (arguments.length === 1 && type(arguments[0]) === 'object') {
-        merge_(arguments[0], model);
+        toMerge = arguments[0];
       } else if (arguments.length === 2 &&
-        type(arguments[0]) === 'string') {
+          type(arguments[0]) === 'string') {
```

Comment:
Nit: I think this can just be put on the line above without going over the 80 line limit, no?

Reply:
Right, moved it